File: pyg-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {


    this.findPage=function (page,rows) {
        return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    };

    this.add=function (entity) {
        return  $http.post('../brand/add.do',entity);
    };

    this.update=function (entity) {
        return  $http.post('../brand/update.do',entity);
    };

    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    };

    this.del=function (selectIds) {
        return  $http.get("../brand/delete.do?ids="+selectIds)
    };

    this.search=function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity)
    };

    this.findBrandList=function () {
        return $http.get('../brand/findBrandList.do')
    }
});

File: pyg-sellergoods-interface/src/main/java/com/pinyougou/content/service/BrandService.java
package com.pinyougou.content.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {
    List<TbBrand> findAll();

    PageResult findPage(int pageNum, int pageSize);

    Result add(TbBrand brand);

    TbBrand findOne(Long id);

    Result update(TbBrand brand);

    void delete(Long[] ids);

    //名字一样的条件查询
    PageResult findPage(TbBrand brand ,int pageNum, int pageSize);

    List<Map> findBrandList();
}
